gplugins/lumerical/tests/test_fdtd_recipe.py

```diff
@@ -1,18 +1,16 @@
 import gdsfactory as gf
-import time
-from gdsfactory.components import straight
-from gplugins.lumerical.FdtdDesignRecipe import FdtdDesignRecipe
-from gdsfactory.generic_tech import LAYER, LAYER_STACK
+from gdsfactory.generic_tech import LAYER
 
+from gplugins.lumerical.recipes.fdtd_recipe import FdtdRecipe
 
 
-if __name__ == "__main__":
+def test_fdtd_recipe():
     c = gf.Component("two_references")
     wr1 = c << gf.components.straight(width=0.6, layer=LAYER.WG)
     wr2 = c << gf.components.straight(width=0.6, layer=LAYER.WG)
     wr2.movey(10)
```

Comment:
might be good to test the `is_fresh()` of these too (create one with a component, run it, check `is_fresh() == True` and then change the component and check `is_fresh() == False`)

Reply:
Done: e9cc257130bf07d26894e29cfb6a058dba281d8d